translations.md

```diff
@@ -0,0 +1,19 @@
+# Translations
+
+Several of our systems are multi-lingual and the functionality for our customers depends upon this fact. While we do not currently support what we could call fully multi-lingual, international and localised applications we should work towards this being an eventuality.
+
+Where internationalisation is available you should endeavour to always provide translations for all text rendered to the customer in each available language. This includes error messages etc which may not be rendered directly within templates.
+
+## What makes a good translation?
+It is recommended that translations be based around entire sentences or phrases and do not include any markup. This is to prevent cases where word order, sentence structure and changing genders, ordinals etc. could create incorrect translations.
+
+## Where can I get translations?
+To get translations for text you can raise a ticket on the [European Translations Kanban board](https://holidayextras.jira.com/secure/RapidBoard.jspa?rapidView=292&projectKey=DEM). When doing so please specify which languages you require the text to be translated into.
```

Comment:
The last time I used this board for German translations I was told to use the email address `translations@holidyaextras.de` instead, although it makes sense to use a single jira board instead